### ISSUE TITLE
Arabic Language Problem

### ISSUE THREAD

**Issue by blade1337a** [arcusmaximus/YTSubConverter#22]
When I'm using shadow effects like "glow" and "soft shadow" before converting the .ass file, it seems that the first word on the right (Arabic is an RTL language) appear way far the rest of the words. However, when I use the "glow" effect alone or the "soft shadow" effect alone it works, but the other combinations aren't working.

**Comment by arcusmaximus** [comment]
Looks fine when I try it (all spaces have the same width):
![test](https://user-images.githubusercontent.com/133129/92321829-6eb87b80-f02d-11ea-8827-e445797dafd4.png)
Could you send your .ass file?

**Comment by arcusmaximus** [comment]
I reproduced the problem in the end. Surprise surprise, it's a YouTube bug: they don't set the padding correctly when the subtitle is right-to-left. The green spacer boxes, which are on the outside in LTR languages, end up on the inside and create a gap.

![image](https://user-images.githubusercontent.com/133129/93026653-f98a0f00-f607-11ea-81d0-905068f74c54.png)
![image](https://user-images.githubusercontent.com/133129/93026636-cf385180-f607-11ea-95aa-7a84b679b5af.png)

YTSubConverter 1.5.1 fixes this by hiding YouTube's padding and adding its own. Your subtitles should now look correct with multiple shadows.

As a bonus, I added a new `\ytdir` tag for setting the subtitle direction to RTL in subtitles for a LTR language. (Not that I think you'll need it)

Thank you for using YTSubConverter and for reporting this!
